docs/sphinx/dev_guide/ci_tasks.rst

```diff
@@ -12,99 +12,142 @@
 
-For example, an entry for a build using a clang compiler with CUDA is:
+For example, an entry for a build using the clang 12.0.1 compiler with CUDA 
+11.5.0 on the LC lassen machine would be something like this:
 
 .. code-block:: bash
 
-  ibm_clang_10_0_1_cuda_10_1_168:
+  clang_12_0_1_cuda_11_5_0:
     variables:
-      SPEC: "+cuda cuda_arch=70 %clang@ibm.10.0.1 ^cuda@10.1.168"
+      SPEC: " ~shared +openmp +tests +cuda cuda_arch=70 %clang@12.0.1 ^cuda@11.5.0"
     extends: .build_and_test_on_lassen
 
-To update, change the corresponding spec item, such as clang compiler
-or version, or cuda version. Then, update the label accordingly.
+Here, we enable OpenMP and CUDA, both of which must be enabled to use them, 
+and specify the CUDA target architecture 'sm_70'.
+
+To add a shared configuration, it must be added to the appropriate
+``<MACHINE>-build-and-test.yml`` file in the 
+`RADIUSS Shared CI <https://github.com/LLNL/radiuss-shared-ci>`_ project.
+Create a branch there, add the job entry, and create a pull request.
 
-It is important to note that the build spec information must reside in 
-the ``compilers.yaml`` and/or ``packages.yaml`` file for the system type
-in the `radiuss-spack-configs <https://github.com/LLNL/RAJA/blob/develop/scripts>`_ submodule. If the desired information is not there,
-try updating the submodule to a newer version. If the information
-is still not available, create a branch in the 
-`RADIUSS Spack Configs <https://github.com/LLNL/radiuss-spack-configs>`_ repo, add the needed spec info, and create a pull request.
+A spec containing the compiler and version information for the machine must 
+exist in the `RADIUSS Spack Configs project <https://github.com/LLNL/radiuss-spack-configs>`_. If it does not exist there, create a branch, add the spec entry,
+and create a pull request. After the PR is merged, update the RADIUSS Spack
+Configs submodule in RAJA to be able to run it in GitLab CI.
 
-.. important:: Build spec information used in RAJA Gitlab CI pipelines
+Modifying a configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To change an existing configuration, change the relevant information in the 
+configuration, such variant(s) or target compute architecture, in the 
+appropriate 
+``RAJA/.gitlab/<MACHINE>-build-and-test-extra.yml`` file. Make sure to 
+also modify the job label, so it is descriptive of the configuration.
+
+To modify a shared configuration, it must be changed in the appropriate
+``<MACHINE>-build-and-test.yml`` file in the 
+`RADIUSS Shared CI <https://github.com/LLNL/radiuss-shared-ci>`_ project.
+Create a branch there, modify the job entry, and create a pull request.
+
+.. important:: Build spec information used in RAJA GitLab CI pipelines
                must exist in the ``compilers.yaml`` file and/or 
                ``packages.yaml`` file for the appropriate system type in
                the `RADIUSS Spack Configs <https://github.com/LLNL/radiuss-spack-configs>`_ repo.
```

Comment:
This is a repetition of the final note of previous section.

```suggestion
```

Reply:
Yup. Thanks. Fixed.